includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -27,7 +27,6 @@
 class AMP_Tag_And_Attribute_Sanitizer extends AMP_Base_Sanitizer {
 
 	const DISALLOWED_TAG                       = 'DISALLOWED_TAG';
-	const DISALLOWED_TAG_MULTIPLE_CHOICES      = 'DISALLOWED_TAG_MULTIPLE_CHOICES';
```

Comment:
I've went ahead and removed the `DISALLOWED_TAG_MULTIPLE_CHOICES` error code as I don't see a meaningful way of displaying the multiple validation errors on the page. Instead, each error will be sanitized separately which allows for each to be viewed.